engineio/src/asynchronous/async_transports/websocket_general.rs

```diff
@@ -42,9 +42,9 @@ impl AsyncWebsocketGeneralTransport {
         let mut sender = self.sender.lock().await;
 
         sender
-            .send(Message::text(Cow::Borrowed(from_utf8(&Bytes::from(
+            .send(Message::text(Utf8Bytes::try_from(Bytes::from(
```

Comment:
This and the other changes in this file are due to upgrading tungstenite. This avoids an extra allocation that was happening inside of the text call.